src/library/librarycontrol.cpp

```diff
@@ -311,14 +311,29 @@ LibraryControl::LibraryControl(Library* pLibrary)
     // Track Color controls
     m_pTrackColorPrev = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "track_color_prev"));
     m_pTrackColorNext = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "track_color_next"));
+    m_pTrackColorSelector = std::make_unique<ControlEncoder>(
+            ConfigKey("[Library]", "track_color_selector"), false);
     connect(m_pTrackColorPrev.get(),
             &ControlPushButton::valueChanged,
             this,
-            &LibraryControl::slotTrackColorPrev);
+            [this](double value) {
+                if (value > 0)
+                    LibraryControl::slotTrackColorSelector(-1);
+            });
```

Comment:
Oh, I didn't notice the clang-tidy warnign in #13066 
> :warning:  statement should be inside braces [readability-braces-around-statements]

Please use braces here and also fix it for the per-track implementation.

Reply:
done